src/basicio.cpp

```diff
@@ -643,8 +643,8 @@ class EXIV2API BlockMap {
 
 void MemIo::Impl::reserve(size_t wcount) {
   const size_t need = wcount + idx_;
-  size_t blockSize = 32 * 1024;  // 32768
-  const size_t maxBlockSize = 4 * 1024 * 1024;
+  auto blockSize = size_t(32) * 1024;  // 32768
```

Comment:
Same thing in this commit. I do not think these clang-tidy changes are bringing anything possitive to the codebase. If something, it is making the code a bit more difficult to read.